tests/README.md

```diff
@@ -1,14 +1,90 @@
-Running and creating tests                        {#running-and-creating-tests}
-==========================
+# Running and creating tests                        {#running-and-creating-tests}
 
 There are a number of tests included in RIOT. They are located in the
 [tests folder](https://github.com/RIOT-OS/RIOT/tree/master/tests). These tests
 allow basic functionality to be verified as well as provide an example of
 usage.
 
+# Running automated tests
 
-Implementing automated tests
-----------------------------
+Some tests can be performed automatically. The test automation scripts are
+defined in the `<test_application>/tests/` folder. They are written in python
+and interact through the uart with the test application code running on a
```

Comment:
```suggestion
and interact through the serial (typically UART) with the test application code running on a
```

Some boards use e.g. USB CDC ACM (a virtual USB modem) for the serial interface by default. The tests use the default serial, which some exceptions that require special handling.

Reply:
It's not my text - by I agree that this change is valuable - as most modern boards use USB for serial.
@maribu - thanks.

I commit this small fix, and squash commits.